multisend-ui/utils/decoder.ts

```diff
@@ -1,5 +1,125 @@
 import { ethers } from 'ethers';
 
+const OPENCHAIN_LOOKUP_URL = 'https://api.openchain.xyz/signature-database/v1/lookup';
```

Comment:
nitpick: ideally all these downstream calls go through the [API](https://github.com/reown-com/safe-decoder/tree/main/multisend-ui/app/api) such that

1) Response format can remain static should we ever switch to another provider
2) API keys are not leaked